app.js

```diff
@@ -408,8 +413,9 @@ function processModule(moduleName) {
             return;
           }
         } else if ((statement.expression.type === 'CallExpression') && (get(statement, 'expression.callee.callee.name') === 'require')) {
-          const arguments = get(statement, 'expression.arguments');
-          if ((arguments.length === 2) && (arguments[0].name === 'self') && (arguments[1].name === 'options')) {
+          const args = get(statement, 'expression.arguments');
```

Comment:
`Parsing error: Binding arguments in strict mode`

Reply:
Oh yeah, that name should be avoided, too magical.